src/main/java/teammates/common/util/Templates.java

```diff
@@ -48,6 +48,12 @@ public static String populateTemplate(String template, String... keyValuePairs)
                 FileHelper.readResourceFile("userEmailTemplate-feedbackSessionPublished.html");
         public static final String USER_FEEDBACK_SUBMISSION_CONFIRMATION =
                 FileHelper.readResourceFile("userEmailTemplate-feedbackSubmissionConfirmation.html");
+        public static final String FRAGMENT_FEEDBACK_SESSION_ACCESS_LINK =
+                FileHelper.readResourceFile("userEmailTemplateFragment-feedbackSessionRecoveryAccessLink.html");
+        public static final String USER_FEEDBACK_SESSIONS_ACCESS_LINKS =
+                FileHelper.readResourceFile("userEmailTemplate-feedbackSessionsRecoveryAccessLinks.html");
+        public static final String USER_FEEDBACK_SESSIONS_ACCESS_LINKS_NONE =
```

Comment:
Oh you mean put recovery in front?

Reply:
Emmmm can you elaborate more here.. For `not returning a list`

Reply:
yes i mean putting recovery in the variable name...and the email template as well...after all you are calling the variables by their names around in the code...so that ppl will know that this is related to the session link recovery...

Reply:
Thank you for the prompt response, and can I ask what is the previous issue about?

Reply:
yes i replied you in the reopened conversation. If you take a look at "File changed" and look for the EmailGenerator file, you will see what I wrote. Seems like no email was sent for replying on reopened resolved conversations...
And yes please check for my comments in all the reopened convos... sorry for the trouble :(